tests/Feature/OfficerTest.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\Officer;
+use App\Models\User;
+use Faker\Factory;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class OfficerTest extends TestCase
+{
+    use RefreshDatabase;
+    public function test_only_admin_can_access_officers()
+    {
+        $user = User::factory()->create();
+
+        $response = $this->actingAs($user)->get(route('admin.officers.index'));
+
+        $response->assertViewIs('pages.officers.index');
+    }
+
+    public function test_fail_if_not_admin_user_try_to_access_officers()
+    {
+        $user = User::factory()->create([
+            'role' => User::$CASHIER
+        ]);
```

Comment:
I think it would be best that you add other roles here apart from cashier only.

Reply:
I added additional tests not just for the cashier and also for all of the users.
**Image evidence**
![image](https://user-images.githubusercontent.com/43285621/134672848-98728bf0-f0af-4e76-930a-326619d242c5.png)